examples/custom-vpc/main.tf

```diff
@@ -4,9 +4,12 @@ resource "random_uuid" "suffix" {
 module "network" {
   source = "../../modules/network"
 
-  suffix          = lower(substr(random_uuid.suffix.id, 0, 5))
-  create_database = true
-  vpc_cidr_block  = "10.0.0.0/18"
+  suffix               = lower(substr(random_uuid.suffix.id, 0, 5))
+  create_database      = true
+  vpc_cidr_block       = "10.0.0.0/18"
+  enable_dns_hostnames = false
+  private_subnet_tags  = {}
+  public_subnet_tags   = {}
```

Comment:
since this is an example code, we might as well define some tags here :)

Reply:
Sure - I just hadn't because the example wasn't really about tagging subnets. I just had to set those properties because they're required for the submodule (but not for the root module).

Reply:
Yeah - it was just a simple way to create a VPC for the example.